javascript/big-practice/src/scripts/views/listNoteView.js

```diff
@@ -39,8 +39,8 @@ export default class ListNoteView {
    * @description if user still note logged in, it will move to login page
    */
   checkUserLoggedIn() {
```

Comment:
The purpose we want when calling this method is to navigate to a page. So please choose another name for this.